### PR TITLE
Mirage models for articles

### PR DESCRIPTION
**Pull request by jonnii**

### CODE REVIEW

**Comment by jonnii** [comment]
@Alonski @alexlafroscia a mirage factory for authors and articles. should be a good start for working on the front page (pagination etc...).

File: mirage/factories/article.js [hunk 1]
```diff
@@ -1,11 +1,30 @@
...
+  author: association(),
 
-  title: 'title',
+  title() {
+    return faker.lorem.words();
+  },
 
-  slug(i) {
-    return 'slug-' + i;
+  tagList: ['dragons', 'training'],
```
Comment (Alonski) [inline mirage/factories/article.js]: Why not generate these? Also so it isn't a fixed number of tags and random tags?

Comment (jonnii) [inline mirage/factories/article.js]: I just didn't get around to it

**Comment by jonnii** [comment]
@Alonski i did something for random tags, pretty low fidelity but should be good enough I think.

**Comment by jonnii** [comment]
@Alonski ok this is rebased, ready to merge!

**Pull request merged**